Xamarin.Forms.Core/NavigationPage.cs

```diff
@@ -184,6 +186,12 @@ public Task PushAsync(Page page)
 
 		public event EventHandler<NavigationEventArgs> Pushed;
 
+		internal static void SetInternalPadding(Page page, Thickness bound)
+		{
+			if(!((Thickness)page.GetValue(InternalPaddingProperty)).Equals(bound))
+				page.SetValue(InternalPaddingProperty, bound);
```

Comment:
@StephaneDelcroix Do you want to add this in the `validateValue' delegate of the BP?

```chsarp
!((Thickness)page.GetValue(InternalPaddingProperty)).Equals(bound)
```

If yes then BP will throw exception if old value equals to new value but this is not error. Just I don't want set new value if it equals old value. Maybe do you know better solution?